app/backend/constants.js

```diff
@@ -80,7 +80,10 @@ const taps = [
     name: 'Redshift',
     tapKey: 'tap-redshift',
     tapImage: 'dataworld/tap-redshift:1.0.0b9',
-    repo: 'https://github.com/datadotworld/tap-redshift'
+    repo: 'https://github.com/datadotworld/tap-redshift',
+    specImplementation: {
+      usesLogBased: false
```

Comment:
Isn't `false` the implicit default? Shouldn't MySQL and Postgres be marked as `usesLogBased: true`?